dati_playground/validators/filename_match_directory.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import re
+from pathlib import Path
+
+log = logging.getLogger(__name__)
+
+# List of filenames to be excluded
+EXCLUDED_FILENAMES = [
+    "index.ttl",
+    "datapackage.json",
+    "context-short.ld.yaml",
+    "rules.shacl",
+    "latest",
+]
+
+# List of extensions to be excluded
+EXCLUDED_EXTENSIONS = [
+    ".md",
+    ".shacl",
+    ".frame.yamlld",
+    ".ld.yaml",
+    ".schema.yaml",
+    ".example.yaml",
+    ".example.ttl",
+    ".png",
+    ".html",
+    ".xml",
+    ".xsd",
+]
+
+
+def validate(fpath: Path, errors):
+
+    if fpath.is_dir():
+        log.debug(f"The dir '{fpath.name}' in path '{fpath}' is not checked")
+        return True
+
+    suffixes = fpath.suffixes
+    extension = "".join(suffixes)
+    filename = re.sub(extension, "", fpath.name)
+
+    if (
+        fpath.name in EXCLUDED_FILENAMES
```

Comment:
Shouldn't this be case insensitive? 

(probably also in other places when part of filenames are checked, I'm keeping this comment as a placeholder for the other instances as well)

Reply:
ok